pages/index.vue

```diff
@@ -368,12 +368,12 @@
               </p>
               <div class="flex justify-center sm:justify-start">
                 <Button
-                  href="https://github.com/ipfs/go-ipfs"
+                  href="https://github.com/ipfs/kubo"
                   target="_blank"
                   rel="noopener"
-                  text="Get go-ipfs"
+                  text="Get Kubo"
                   :on-click="
-                    (item) => onCTAClick({ ui: 'install/go-ipfs', ...item })
+                    (item) => onCTAClick({ ui: 'install/kubo', ...item })
```

Comment:
@johnnymatthews I assume this is ok? (I imagine we want to see if the new name impacted the number of clicks)